utility_network/display-content-of-utility-network-container/src/main/java/com/esri/samples/display_content_of_utility_network_container/DisplayContentOfUtilityNetworkContainerController.java

```diff
@@ -0,0 +1,305 @@
+                  vBox.setVisible(true);
+
+                  // fetch the features from the elements
+                  ListenableFuture<List<ArcGISFeature>> fetchFeaturesFuture = utilityNetwork.fetchFeaturesForElementsAsync(contentElements);
+                  fetchFeaturesFuture.addDoneListener(() -> {
+                    try {
+                      // get the content features and give them each a symbol, and add them as a graphic to the graphics overlay
+                      List<ArcGISFeature> contentFeatures = fetchFeaturesFuture.get();
+                      contentFeatures.forEach(content -> {
+                        Symbol symbol = content.getFeatureTable().getLayerInfo().getDrawingInfo().getRenderer().getSymbol(content);
+                        graphicsOverlay.getGraphics().add(new Graphic(content.getGeometry(), symbol));
+                      });
+
+                      Geometry firstGraphic = graphicsOverlay.getGraphics().get(0).getGeometry();
+                      double containerViewScale = containerElement.getAssetType().getContainerViewScale();
+
+                      if (graphicsOverlay.getGraphics().size() == 1 && firstGraphic instanceof Point) {
+                        mapView.setViewpointCenterAsync((Point) firstGraphic, containerViewScale).addDoneListener(() -> {
+
+                          // the bounding box, which defines the container view, may be computed using the extent of the features
+                          // it contains or centered around its geometry at the container's view scale
+                          Geometry boundingBox = mapView.getCurrentViewpoint(Viewpoint.Type.BOUNDING_GEOMETRY).getTargetGeometry();
+                          identifyAssociationsWithExtent(boundingBox);
+                          new Alert(Alert.AlertType.INFORMATION, "This feature has no associations").show();
+
+                        });
+
+                      } else {
+                        Geometry boundingBox = GeometryEngine.buffer(graphicsOverlay.getExtent(), 0.05);
+                        identifyAssociationsWithExtent(boundingBox);
+                      }
+
+                    } catch (Exception ex) {
+                      new Alert(Alert.AlertType.ERROR, "Error fetching features for elements.").show();
+                      ex.printStackTrace();
+                    }
+                  });
+
+                } else {
+                  new Alert(Alert.AlertType.ERROR, "No content elements found").show();
+                }
+
+              } catch (Exception ex) {
+                new Alert(Alert.AlertType.ERROR, "Error getting containment associations.").show();
+                ex.printStackTrace();
+              }
+            });
+          }
+
+        } catch (InterruptedException | ExecutionException ex) {
+          new Alert(Alert.AlertType.ERROR, "Error getting result of the query.").show();
+          ex.printStackTrace();
+        }
+      });
+
+    }
+
+  }
```

Comment:
```suggestion
      });
    }
  }
```
Could lose a few line spaces here